src/api/app/views/webui/project/show.html.haml

```diff
@@ -86,5 +86,8 @@
                                                               confirmation_text: "Please confirm deletion of the project '#{@project}'",
                                                               action: project_destroy_path(project: @project) })
 
+- if policy(Report.new(reportable: @project).create?
```

Comment:
I think we should move this into `ReportComponent.render`..

Reply:
Hm, yeah, I'll write a PR later to move the policy checks into the component. Works for you?

Reply:
sure, where do you track this?

Reply:
Here: https://trello.com/c/lRN12PZK/31-move-the-policy-checks-inside-the-report-component

Reply:
Well, that check is controlling if the "button" to report content shows or not, so either we create a reporting button component and move the check there, or moving it inside the `ReportComponent.render` won't work...

Reply:
Alright, we'll move the one affecting the component